src/Components/VitalsMonitor/HL7PatientVitalsMonitor.tsx

```diff
@@ -40,7 +43,12 @@ export default function HL7PatientVitalsMonitor(props: IVitalsComponentProps) {
   }, [props.socketUrl]);
 
   const bpWithinMaxPersistence = !!(
-    (data.bp?.["date-time"] && isWithinMinutes(data.bp?.["date-time"], 30)) // Max blood pressure persistence is 30 minutes
+    data.bp?.["date-time"] &&
+    props.patientCurrentBedAssignmentDate &&
+    isWithinMinutes(
+      data.bp?.["date-time"],
+      props.patientCurrentBedAssignmentDate
+    )
```

Comment:
Have a look at https://day.js.org/docs/en/query/is-after

```suggestion
    dayjs(data.bp?.["date-time"]).isAfter(props.patientCurrentBedAssignmentDate)
```